ProGetAutomation/Functions/Remove-ProGetAsset.ps1

```diff
@@ -0,0 +1,41 @@
+function Remove-ProGetAsset
+{
+    <#
+        .SYNOPSIS
+        Used to remove assets from the proget asset manager. 

```

Comment:
Fix case of `proget` to `ProGet`. Change `Used to remove` to `Removes`.

Reply:
Done